test/pageobjects/Confirm.js

```diff
@@ -0,0 +1,87 @@
+import Base from './BasePage.js'
+import { verifyElementCopy } from '../utils/mochaw'
+
+class Confirm extends Base {
+  get title() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
+  get makeSureClearDetailsMessage() { return this.$('.onfido-sdk-ui-PageTitle-subTitle')}
+  get redoBtn() { return this.$('.onfido-sdk-ui-Confirm-retake')}
+  get confirmBtn() { return this.$('.onfido-sdk-ui-Confirm-btn-primary')}
+  get uploaderError() { return this.$('.onfido-sdk-ui-Uploader-error')}
+  get errorTitleText() { return this.$('.onfido-sdk-ui-Error-title-text')}
+  get errorTitleIcon() { return this.$('.onfido-sdk-ui-Error-title-icon-error')}
+  get warningTitleIcon() { return this.$('.onfido-sdk-ui-Error-title-icon-warning')}
+  get errorInstruction() { return this.$('.onfido-sdk-ui-Error-instruction-text')}
+  get uploadedVideo() { return this.$('.onfido-sdk-ui-Confirm-CaptureViewer-video')}
+
+  async verifyCheckReadabilityMessage(copy) {
+    const confirmStrings = copy.confirm
+    verifyElementCopy(this.title, confirmStrings.document.title)
+  }
+
+  async verifyMakeSurePassportMessage(copy) {
+    const confirmStrings = copy.confirm
+    verifyElementCopy(this.makeSureClearDetailsMessage, confirmStrings.passport.message)
+  }
+
+  async verifyMakeSureDrivingLicenceMessage(copy) {
+    const confirmStrings = copy.confirm
+    verifyElementCopy(this.makeSureClearDetailsMessage, confirmStrings.driving_licence.message)
+  }
+
+  async verifyMakeSureIdentityCardMessage(copy) {
+    const confirmStrings = copy.confirm
+    verifyElementCopy(this.makeSureClearDetailsMessage, confirmStrings.national_identity_card.message)
+  }
+
+  async verifyNoDocumentError(copy) {
+    const confirmErrorStrings = copy.errors
+    verifyElementCopy(this.errorTitleText, confirmErrorStrings.invalid_capture.message)
+    this.errorTitleIcon.isDisplayed()
+    verifyElementCopy(this.errorInstruction, confirmErrorStrings.invalid_capture.instruction)
+  }
+
+  async verifyFileSizeTooLargeError(copy) {
+    const confirmErrorStrings = copy.errors
+    verifyElementCopy(this.uploaderError, confirmErrorStrings.invalid_size.message + ". " + confirmErrorStrings.invalid_size.instruction + ".")
```

Comment:
Wouldn't it be better here and at line 50 for the punctuation to be in the message/instruction text itself? 🤔

Reply:
Interesting... I didn't notice it. I just copy pasted the tests from the former `happy.js`. I wonder why the dots are not in the copy itself. I will have a look.